ui/core/src/model/units/throughput.ts

```diff
@@ -82,6 +88,12 @@ export const THROUGHPUT_UNIT_CONFIG: Readonly<Record<ThroughputUnit, UnitConfig>
 };
 
 export function formatThroughput(value: number, { unit, shortValues, decimalPlaces }: ThroughputFormatOptions): string {
+  // special case for data throughput
+  if (unit == 'bytes/sec') {
```

Comment:
NB: 'a bit quick-win to manage it like this but imho that's ok, we (sh|w)ould refactor this part later on when adding new "data rate" units